tests/python_tests/test_static_llm_pipeline.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import openvino_genai as ov_genai
+from openvino.runtime import Core
+import pytest
+import math
+from ov_genai_test_utils import (
+    get_models_list,
+    get_chat_models_list,
+)
+
+
+# This test suite is designed specifically to validate the functionality and robustness of the StaticLLMPipeline on CPU.
+# On machines without NPU, the pipeline will automatically fall back to CPU, which aligns with the goal of these tests.
+# However, on machines where an NPU is available, the tests may fail due to NPU-specific compile or runtime issues,
+# which are outside the scope of this test suite and not its intended focus.
+# is_npu_available = 'NPU' in Core().get_available_devices()
+is_npu_available = False
```

Comment:
Is it possible to create Core here and check if `NPU` is in available devices?

Reply:
Oh Jez, it's hardcode, sorry will fix

Reply:
Removed this part completely, now fallback happens via config even on the platform with NPU